makeflow/src/dag_node.c

```diff
@@ -346,8 +346,15 @@ struct jx * dag_node_env_create( struct dag *d, struct dag_node *n )
 	while((key = set_next_element(d->export_vars))) {
 		char *value = dag_variable_lookup_string(key, &s);
 		if(value) {
+			if(strcmp(key, "CORES")) {
```

Comment:
strcmp returns 0 if the comparison succeeds. Here you are setting OMP_NUM_THREADS to the value of each other variable, but CORES.